data-catalog-entry-schema.json

```diff
@@ -0,0 +1,263 @@
+      "items": {
+        "type": "object",
+        "properties": {
+          "project_name": {
+            "title": "Project Name",
+            "description": "This is the name of your server folder if it is a long term project. It is a drop down menu which needs to get updated as projects are added.",
+            "type": "string",
+            "enum": [
+              "food-for-thought",
+              "baboon",
+              "bat_foraging",
+              "bci_general",
+              "human_network_coordination",
+              "blackbuck",
+              "hyena",
+              "bonobo",
+              "kinkajou_cognition",
+              "lion",
+              "meerkat",
+              "capuchin_bci",
+              "mpala_general",
+              "capuchin_coiba",
+              "ngogo_monkey",
+              "capuchin_lomas",
+              "orangutan",
+              "capuchin_santarosa",
+              "ccas",
+              "red_colobus",
+              "cichlid",
+              "sheep",
+              "coati",
+              "sifaka_fossa",
+              "dolphin_human",
+              "social_foraging_bci",
+              "drone",
+              "tuanan_orangutan",
+              "wilddog"
+            ]
+          },
+          "server_address": {
+            "title": "Server Address",
+            "description": "location on server where data catalog entry is stored",
+            "type": "string"
+          }
+        }
+      }
+    },
+    "people": {
+      "title": "People",
+      "description": "People involved in project. Make an an entry for each person.",
+      "type": "array",
+      "items": {
+        "type": "object",
+        "properties": {
+          "name": {
+            "title": "Name",
+            "description": "researcher name",
+            "type": "string"
+          },
+          "institute": {
+            "title": "Institute",
+            "description": "institutional affiliation(s)",
+            "type": "string"
+          },
+          "email": {
+            "title": "Email",
+            "anyOf": [
+              {
+                "type": "string",
+                "format": "email"
+              },
+              {
+                "type": "string",
+                "maxLength": 0
+              }
+            ]
+          },
+          "role": {
+            "title": "Role",
+            "description": "What is role in project (dropdown menu)",
+            "type": "string",
+            "enum": [
+              "PI",
+              "PhD Student",
+              "Masters Student",
+              "Collaborator",
+              "HIWI",
+              "Field Assistant",
+              "Lab Assistant",
+              "Analyst"
+            ]
+          },
+          "uploader": {
+            "title": "Uploader",
+            "description": "whether they are responsible for data catalog entry upload.",
+            "type": "boolean",
+            "default": false
+          }
```

Comment:
`date` as a property of person is listed in the examples but not the shiny app. lmk if it should be included.

Reply:
I think we decided no on this.